foundation-extensions/mobile-core/acpcore-aepcore.md

```diff
@@ -13,6 +13,8 @@ This document is a reference comparison of ACPCore \(2.x\) APIs against their eq
 
 ### trackAction
 
+Actions are events that occur in your application. You can use the `trackAction` method to track and measure an action. Each action has one or more corresponding metrics that are incremented each time the event occurs. For example, you can use an action to track new subscriptions, every time an article is viewed, or every time a level is completed.
```

Comment:
Do we need in detail description for all these APIs in the migration guide? We usually cover these details in the API reference page, the migration guide just provides a quick comparison between ACP/AEP APIs syntax